glide.yaml

```diff
@@ -32,4 +32,6 @@ import:
   version: ~1.1
 - package: github.com/arschles/assert
   version: 6882f85ccdc7c1822b146d1a6b0c2c48f91b5140
-- package: github.com/minio/minio-go
+- package: github.com/docker/distribution
+  repo: https://github.com/kmala/distribution
```

Comment:
ok, thanks for the pointer. I just gave a review there. I'll add that PR to the required list for this one.